lightningd/plugin.c

```diff
@@ -1086,9 +1086,24 @@ void json_add_opt_plugins(struct json_stream *response,
 			  const struct plugins *plugins)
 {
 	struct plugin *p;
+	struct plugin_opt *opt;
+	const char *name;
+
+	// we output 'plugins' and their config as substructure:
+	// "plugins" : { "name" -> { "param" : "value" }, ... }
```

Comment:
We usually use `/* ... */` style comments.

Reply:
Yes sorry, I almost ever miss that!